benchmark_runner/main/environment_variables.py

```diff
@@ -214,5 +228,24 @@ def environment_variables_dict(self, value: dict):
         """
         self._environment_variables_dict = value
 
+    def workload_type(self, workload: str):
+        """
+        Return the workload type for a given workload
+        """
+        if workload in self._environment_variables_dict['workloads'] and workload.split('_')[0] in self._environment_variables_dict['workload_types']:
+            return self._environment_variables_dict['workload_types'][workload.split('_')[0]]
+        else:
+            return None
+
+    def workload_path(self, workload:str):
```

Comment:
The method name should be get_workload_flavor_path and also please add method document